examples/platforms/cc2652/CMakeLists.txt

```diff
@@ -66,33 +76,19 @@ add_library(openthread-cc2652
 
 set_property(TARGET openthread-cc2652 PROPERTY C_STANDARD 99)
 
-target_link_libraries(openthread-cc2652
-    PUBLIC
-        cc13x2-cc26x2-driver
-    PRIVATE
-        ${OT_MBEDTLS}
-        ot-config
+target_link_libraries(
+    openthread-cc2652
+    PUBLIC cc13x2-cc26x2-driver
+    PRIVATE ${OT_MBEDTLS} ot-config
```

Comment:
I think the old style is better.

Reply:
Yes, I agree. But I think the new style is also acceptable. I don't think there will be a perfect tool for us to format code. This style just make sure the style is consistent. I think a consistency style is more important.